stacks/apps-300A.yml

```diff
@@ -151,11 +155,15 @@ Resources:
         VpcPrivateSubnet1Id: !Ref VpcPrivateSubnet1Id
         VpcPrivateSubnet2Id: !Ref VpcPrivateSubnet2Id
         VpcPrivateSubnet3Id: !Ref VpcPrivateSubnet3Id
-        MetricsKinesisStreamArn: !Sub /prx/${EnvironmentTypeAbbreviation}/analytics-ingest-lambda/METRICS_KINESIS_STREAM
+        MetricsKinesisStreamArn: !Sub /prx/${EnvironmentTypeAbbreviation}/analytics-ingest-lambda/METRICS_KINESIS_STREAM # TODO: remove
```

Comment:
It'd be nice if these names had some mention of Dovetail in them as they're being passed around between templates.

Reply:
This one specifically is being deprecated - so i didn't change their names.

The new ones are below - `AnalyticsMetricsKinesisStreamArn` and `AnalyticsDynamoDBKinesisStreamArn`.  I could rename those to start with `Dovetail` instead of `Analytics` though?